README.md

```diff
@@ -77,31 +86,52 @@ We could simply treat it as the tree
+JavaScript exposes a few ways of finding DOM nodes more or less directly,
+courtesy of the `document` object.
 
 #### `document.getElementById()`
 
-This method provides the quickest access to a node, but it requires that we know something very specific about it — its `id`. Since IDs must be unique, this method only returns one element. (If you have two elements with the same ID, this method returns the first one — keep your IDs unique!) Given the following DOM tree
+This method provides the quickest access to a node, but it requires that we know
+something very specific about it — its `id`. Since IDs must be unique, this
+method only returns one element. (If you have two elements with the same ID,
+this method returns the first one — keep your IDs unique!) Given the following
+DOM tree
 
 ``` html
 <div>
   <h5 id="greeting">Hello!</h5>
 </div>
 ```
 
-we could find the `h5` element with `document.getElementById('greeting')`. Notice how the `id` that we pass to `getElementById` is identical to the `id` in `<h5 id="greeting">`. We can assign properties to HTML nodes (or elements) simply by including them between the `<>` tags at the start of the element (so not in the `</h5>` tag, in this case).
+we could find the `h5` element with `document.getElementById('greeting')`.
+Notice how the `id` that we pass to `getElementById` is identical to the `id` in
+`<h5 id="greeting">`. We can assign properties to HTML nodes (or elements)
+simply by including them between the `<>` tags at the start of the element (so
+not in the `</h5>` tag, in this case).
 
 **Try it out!**
 
-Open up your web inspector (command+option+j on OS X) and find an element on the page — make note of its `id`. Then open up your console, type `document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy DOM node. Try changing a few of its properties!
+Open up your web inspector (command+option+j on OS X) and find an element on the
+page — make note of its `id`. Then open up your console, type
+`document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy
+DOM node. Try changing a few of its properties!
 
 #### `document.getElementsByClassName()`
 
-This method, as its name implies, finds elements by their `className`. Unlike `id`, `className` does not need to be unique; as such, this method returns an HTMLCollection (basically a list of DOM nodes — note that it is _not_ an array, even though it has a `length` property) of all the elements with the given class. You can iterate over an HTMLCollection with a simple `for` loop.
+This method, as its name implies, finds elements by their `className`. Unlike
+`id`, `className` does not need to be unique; as such, this method returns an
+HTMLCollection (basically a list of DOM nodes — note that it is _not_ an array,
+even though it has a `length` property) of all the elements with the given
+class. You can iterate over an HTMLCollection with a simple `for` loop.
```

Comment:
I *do* think we need to talk about `[]` reference of individual elements here....so that we can link getting one node versus getting n-nodes unto one node.